cows/beavis.zen.cow

```diff
@@ -1,6 +1,7 @@
 ##
 ## Beavis, with Zen philosophy removed.
 ##
+$eye = chop($eyes);
```

Comment:
I don't think this will work. There's no `chop` functionality. I'm using an incredibly simple templating language, so any string `$eyes` is replaced with the configured eyes. Check [this](https://github.com/piuccio/cowsay/blob/master/lib/replacer.js#L9-L14)

You need to add something at the bottom that does the chop.

Maybe something as simple as `.replace(/\$eye/g, eyes.charAt(0))` would do the trick

Reply:
Ah, I just noticed this change request minutes after I submitted another pull request https://github.com/piuccio/cowsay/pull/43

Reply:
Removed from all cowfiles in https://github.com/piuccio/cowsay/pull/39/commits/122fcb7996bb472f971643b6a0d5d48aa0a0e29f
```
$eye = chop($eyes);
```